### PR TITLE
3-Tier: End threeTierCheckoutV3 and put variantFixed to 100% of audience

### PR DESCRIPTION
**Pull request by GHaberis**
## What are you doing in this PR?

End threeTierCheckoutV3 and put variantFixed to 100% of audience.

- End and remove `threeTierCheckoutV3` AB test
- Rename `pages/supporter-plus-landing/setup/threeTierABTest.ts` to `pages/supporter-plus-landing/setup/threeTierChecks.ts` 
- Rename export in `threeTierChecks.ts` module to `showThreeTierCheckout` to `threeTierCheckoutEnabled`.
- Update logic in `threeTierCheckoutEnabled` to stop looking for `threeTierCheckoutV3` AB test participations and instead check whether user is in the `patronsOneOffOnly` or  `supporterPlusOnly` AB tests, or is in `countriesAffectedByVATStatus`. If any of these conditions are `true` then `threeTierCheckoutEnabled` returns `false`
- Update `threeTierConfig` to remove variable price config.
- On `weeklyCheckoutForm.tsx` use presence of `threeTierCreateSupporterPlusSubscription` query string param as indication to render 3-tier experience.

**supporterPlusOnly (locally):** https://support.thegulocal.com/uk/contribute?acquisitionData={%20%22abTest%22:{%20%22name%22:%20%22supporterPlusOnly%22,%20%22variant%22:%20%22variant%22%20}%20}

**patronsOneOffOnly (locally):** https://support.thegulocal.com/uk/contribute?acquisitionData={%20%22abTest%22:{%20%22name%22:%20%22patronsOneOffOnly%22,%20%22variant%22:%20%22variant%22%20}%20}

[**Trello Card**](https://trello.com/c/wQnGJv4P/680-end-test-3-on-3-tiers)

### CODE REVIEW

File: support-frontend/assets/components/priceCards/priceCards.tsx [hunk 1]
```diff
@@ -109,9 +107,7 @@ export function PriceCards({
 							alternateLabel={`${simpleFormatAmount(
 								currencies[currency],
 								amount,
-							)}${
-								paymentInterval ? amountIntervalSeperator + paymentInterval : ''
-							}`}
+							)}${paymentInterval ? 'per' + paymentInterval : ''}`}
```
Comment (paul-daniel-dempsey) [inline support-frontend/assets/components/priceCards/priceCards.tsx:110]: Should `per` have a few spaces around it? ie ` per `

Comment (GHaberis) [reply support-frontend/assets/components/priceCards/priceCards.tsx:110]: Yeah good catch! 🎣

File: support-frontend/assets/pages/supporter-plus-landing/setup/threeTierChecks.ts [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+	abParticipations: Participations,
+	countryId: IsoCountry,
+): boolean => {
+	const displayPatronsCheckout = !!abParticipations.patronsOneOffOnly;
+	const displaySupportPlusOnlyCheckout = !!abParticipations.supporterPlusOnly;
+
+	return !(
+		displayPatronsCheckout ||
+		displaySupportPlusOnlyCheckout ||
+		countriesAffectedByVATStatus.includes(countryId)
```
Comment (paul-daniel-dempsey) [inline support-frontend/assets/pages/supporter-plus-landing/setup/threeTierChecks.ts:15]: Thats a neat way of checking once removed AbTest

File: support-frontend/assets/pages/weekly-subscription-checkout/components/weeklyCheckoutForm.test.tsx [hunk 3]
```diff
@@ -96,7 +96,7 @@ describe('Guardian Weekly checkout form', () => {
 					countryId: 'GB',
 					currencyId: 'GBP',
 				},
-				abParticipations: [],
+				abParticipations: {},
```
Comment (paul-daniel-dempsey) [inline support-frontend/assets/pages/weekly-subscription-checkout/components/weeklyCheckoutForm.test.tsx:99]: Maybe this is an array as `abParticipations` plural? Don't think it make s a difference in the test to be honest.

Comment (GHaberis) [reply support-frontend/assets/pages/weekly-subscription-checkout/components/weeklyCheckoutForm.test.tsx:99]: `abParticipations` is actually an object and not an array, the original mock state was incorrect, it didn't break anything but I've updated it nevertheless.

Comment (GHaberis) [reply support-frontend/assets/pages/weekly-subscription-checkout/components/weeklyCheckoutForm.test.tsx:99]: I was looking at this file as the tests did actually flag a regression I would've introduced if they hadn't caught it!

File: support-frontend/assets/pages/weekly-subscription-checkout/components/weeklyCheckoutForm.tsx [hunk 4]
```diff
@@ -199,7 +198,9 @@ function WeeklyCheckoutForm(props: PropTypes) {
 		props.setBillingCountry(props.deliveryCountry);
 	};
 
-	const inThreeTier = showThreeTierCheckout(props.participations);
+	const urlParams = new URLSearchParams(window.location.search);
```
Comment (paul-daniel-dempsey) [inline support-frontend/assets/pages/weekly-subscription-checkout/components/weeklyCheckoutForm.tsx:201]: Ah I see, no abTest definition so need to rely upon URL Param. Probably should of been like this during the test, oh well, thanks!

**Review by paul-daniel-dempsey — APPROVED** [review]
Looks very good, nice and neat `threeTierCheck` especially, PR runs perfectly. Thanks.

**Review by jamesgorrie — APPROVED** [review]
Neato!

**Pull request merged**